src/layers/PolygonSeries.js

```diff
@@ -111,10 +111,14 @@ export class PolygonSeries extends PaletteMixin(CoverageMixin(L.Layer)) {
     if (t.coordPref == undefined) {
       t.idx = t.coord = undefined
     } else {
-      let vals = this.domain.axes.get('t').values.map(v => v.getTime())
-      t.idx = indexOfNearest(vals, t.coordPref.getTime())
+      let vals = this.domain.axes.get('t').values.map(v => asTime(v))

```

Comment:
Looks good. Definitely broken before. The same code is also in PointSeries which should be fixed as well.